### PR TITLE
Pgd4k/improvements

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

- Added a few minor fixes/improvements
- Tested implementing the usage of environment variables for ease of use 
- Fixed links to be relative

### CODE REVIEW

**Review by djw-m — APPROVED** [review]
LGTM

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx [hunk 1]
```diff
@@ -27,29 +27,79 @@ helm upgrade --dependency-update \
...
 After assembling the command with the required images, 
-see [Installation](/postgres_distributed_for_kubernetes/latest/installation_upgrade/) for instructions on how to add the repository, 
+see [Installation](../installation_upgrade) for instructions on how to add the repository, 
 deploy the images, and create a certificate issuer. 
 
-## Example 
+For more information about how to assemble your command, see the examples: 
 
-The example command: 
+## Examples
```
Comment (jsilvela) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx:44]: this seems like a sub-section of "Assembling your command". If so, should be an h3 with three `#`.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx [hunk 2]
```diff
@@ -27,29 +27,79 @@ helm upgrade --dependency-update \
...
-* Uses PGD 5.4.1 as the Postgres Distributed version.
-* Uses 5.4.0 as the PGD Proxy version.
+These example commands: 
+
+* Pull images from the `k8s_enterprise_pgd` repository. 
+* Use EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
+* Use PGD 5.4.1 as the Postgres Distributed version.
+* Use 5.4.0 as the PGD Proxy version.
+
+### Example 1: Setting the environment variables before you run the deployment command
```
Comment (jsilvela) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx:53]: as before, if the Examples section was a subsection, then this should be an h4...

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx [hunk 3]
```diff
@@ -27,29 +27,79 @@ helm upgrade --dependency-update \
...
+1.  Set the environment variable to pull images from the `k8s_enterprise_pgd` repository: 
+
+    ```
+    export REPOSITORY_NAME=k8s_enterprise_pgd
+    ```
+
+1.  Set the environment variable to use [your personal token](private_registries/#how-to-retrieve-the-token). In this example, the token is not real:
+
+    ```
+    export EDB_SUBSCRIPTION_TOKEN=aaAa1bbbBBcccccC1a1111bB1C1aBbB
```
Comment (jsilvela) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx:66]: seeing as people tend to blaze through instructions, I'd skip the warning of non-realness, and I'd make the token obviously not real.
```suggestion
1.  Set the environment variable to use [your personal token](private_registries/#how-to-retrieve-the-token):

    ```
    export EDB_SUBSCRIPTION_TOKEN=<my-token-for-the-registry>
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx [hunk 4]
```diff
@@ -27,29 +27,79 @@ helm upgrade --dependency-update \
...
+      --set config.data.PGD_PROXY_IMAGE_NAME=docker.enterprisedb.com/${REPOSITORY_NAME}/${PGD-PROXY}
+      --set image.imageCredentials.username=${REPOSITORY_NAME} \
+      --set image.imageCredentials.password=${EDB_SUBSCRIPTION_TOKEN}
+    ``` 
+
+### Example 2: Replacing the placeholders manually
 
 !!! Note Token 
-    Before running the command, replace the token placeholder.
+    Before running the command, replace the `EDB_SUBSCRIPTION_TOKEN` placeholder.
```
Comment (jsilvela) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx:102]: in the previous example, you set the EDB_SUBSCRIPTION_TOKEN as an env variable.
I'd be consistent in the approach.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/installation_upgrade.mdx [hunk 5]
```diff
@@ -34,6 +34,22 @@ helm repo add edb \
...
+1.  Set `REPOSITORY_NAME` to the name of the repository. In this example, the images come from the `k8s_enterprise_pgd` repository:
+
+    ```
+    export REPOSITORY_NAME=k8s_enterprise_pgd
+    ```
+
+1.  Set `EDB_SUBSCRIPTION_TOKEN` to use your personal token. In this example, the token is not real:
+
+    ```
+    export EDB_SUBSCRIPTION_TOKEN=aaAa1bbbBBcccccC1a1111bB1C1aBbB
```
Comment (jsilvela) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/installation_upgrade.mdx:50]: as my previous comment, I'd suggest setting a clearly-bogus token.

**Review by jsilvela — CHANGES_REQUESTED** [review]
Mostly optional comments, but the headings for the examples should be revised.

**Review by jsilvela — APPROVED** [review]

**Pull request merged**